monkey/infection_monkey/system_info/windows_info_collector.py

```diff
@@ -61,12 +61,15 @@ def get_wmi_info(self):
         LOG.debug('finished get_wmi_info')
 
     def get_mimikatz_info(self):
-        mimikatz_collector = MimikatzCollector()
-        mimikatz_info = mimikatz_collector.get_logon_info()
-        if mimikatz_info:
-            if "credentials" in self.info:
-                self.info["credentials"].update(mimikatz_info)
-            self.info["mimikatz"] = mimikatz_collector.get_mimikatz_text()
-            LOG.info('Mimikatz info gathered successfully')
-        else:
-            LOG.info('No mimikatz info was gathered')
+        LOG.info("Gathering mimikatz info")
+        try:
+            credentials = WindowsCredentialCollector.get_creds()
+            if credentials:
+                if "credentials" in self.info:
+                    self.info["credentials"].update(credentials)
+                self.info["mimikatz"] = credentials
+                LOG.info('Mimikatz info gathered successfully')
+            else:
+                LOG.info('No mimikatz info was gathered')
+        except Exception as e:
+            LOG.info(f"Pypykatz failed: {e}")
```

Comment:
Replace Mimikatz with pypykatz (or with a generic name that's not tied to a specific implementation) in the name, log, and info field name :)

Reply:
Improving naming consistency is a valid point. Regarding renaming mimikatz to pypykatz - on high level it makes most sense to keep mimikatz, because it's well known and it's purpose is clear. On caller level (actual commented code) we don't care how mimikatz cred gathering is implemented, via dll or some kind of python module.